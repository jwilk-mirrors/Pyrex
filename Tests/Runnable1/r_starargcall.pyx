def spam(a, b, c):
	print "Args:", a, b, c

def eggs():
	spam(*(1,2,3))
	spam(*["buckle","my","shoe"])

