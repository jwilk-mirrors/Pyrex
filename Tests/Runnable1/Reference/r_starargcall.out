Args: 1 2 3
Args: buckle my shoe
